osgar/drivers/kloubak.py

```diff
@@ -335,7 +343,7 @@ def slot_can(self, timestamp, data):
                     desired_angle = compute_desired_angle(self.desired_speed, self.desired_angular_speed)
                     angle = joint_rad(self.last_join_angle)
                     diff = abs(desired_angle - angle)
-                    speed = self.desired_speed * (1 - diff/math.radians(AD_MAX_DEG))
+                    speed = self.desired_speed * (1 - diff/math.radians(80))
```

Comment:
this is probably mistake and should be original version

Reply:
No, I see that the name "AD_MAX_DEG" is  slightly confusing. Now it is 45 deg but the maxim angle is still around 80 deg. Maybe we should rename the constant because i don't plan use the maxim angle for calibration anymore.